classes/ImageManager.php

```diff
@@ -688,6 +699,14 @@ public static function write($type, $resource, $filename)
 
                 break;
 
+            case 'avif':
+                if (function_exists('imageavif')) {
+                    $quality = ($avifWebpQuality === false ? 80 : $avifWebpQuality);
+                    $success = imageavif($resource, $filename, $quality);
+                }
```

Comment:
```suggestion
                } else {
                    $success = false;
                }
```
I think you need this to make PHPStan happy lower (and mostly to make sure the `$success` variable is always defined)